### ISSUE TITLE
Login in firefox blocked by validation

### ISSUE THREAD

**Issue by fimdomeio** [RocketChat/Rocket.Chat#1433]
When I try to login to rocket.chat in Firefox I get a message on the top left corner saying "Please enter an email address". My best guess is that this is being cause by an hidden input of type email and that the parent form tag is missing the novalidate atribute

**Comment by geekgonecrazy** [comment]
Did you ever have it save your login information?  Does this work from like in private browsing in firefox?

**Comment by fimdomeio** [comment]
Yes. Everything works correctly on private browsing.
Removed saved password everything working correctly. firefox asked again to save my password. In the information to be saved it says that it will save the password no username. It keeps working when I leave the username empty or when I add my email to it, but it never prefills the username on the login page.

I was guessing this happened when I registered and firefox askeded me for the first time if I wanted to saved the password, but even after cleaning all the passwords and registering a new account I can't replicate the bug. Maybe because the first user goes through a different process.
